src/shared/constants/regions.ts

```diff
@@ -0,0 +1,60 @@
+export const convertProviderSymbol = (sym: string) => symbolProviderMap[sym]
+
+const symbolProviderMap = {
+  zuora: 'Zuora Z-Commerce Platform',
```

Comment:
z-commerce or e-commerce? I genuinely don't know

Reply:
Launch announcement says `Z-Commerce` (but is old).
https://www.zuora.com/press-release/zuora-introduces-the-business-cloud-with-launch-of-the-z-commerce-platform/

Docs just say `Commerce`.
https://knowledgecenter.zuora.com/Commerce

Reply:
changed it to Commerce

Reply:
I'd run it by the e-commerce team just to be sure about the wording, since it might not be open expressed that payments are processed through Zuora

Reply:
Relying on quartz API as the src of truth. Removing this pretty print conversion.